rundeckapp/repository/grails-app/init/repository/BootStrap.groovy

```diff
@@ -1,11 +1,17 @@
 package repository
 
 class BootStrap {
-
+    RepositoryPluginService repositoryPluginService
     def grailsApplication
 
     def init = { servletContext ->
-        log.debug("Repository enabled: " + grailsApplication.config.rundeck.features.repository.enabled)
+        boolean enabled = grailsApplication.config.rundeck.features.repository.enabled in [true, 'true']
+        boolean sync = grailsApplication.config.rundeck.features.repository.syncOnBootstrap in [true,'true']
+        log.debug("Repository enabled: " + enabled)
+        if(enabled && sync) {
+            log.debug("Syncing installed plugins to this server")
```

Comment:
Perhaps "syncing installed plugins" log message should be set to Info level.

Reply:
Log level updated